src/App.tsx

```diff
@@ -15,36 +16,110 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum ESortType {
+  ALPHABETICALLY = 'alphabetically',
+  LENGTH = 'by length',
+  DEFAULT = '',
+}
+
+interface IOptions {
+  sortBy: ESortType;
+  isReversed: boolean;
+}
+
+const getGoods = (goods: string[], { sortBy, isReversed }: IOptions) => {
+  let visibleGoods = [...goods];
+
+  if (isReversed) {
+    visibleGoods = visibleGoods.toReversed();
+  }
+
+  if (sortBy) {
+    switch (sortBy) {
+      case ESortType.ALPHABETICALLY:
+        visibleGoods = visibleGoods.toSorted();
+        if (isReversed) {
+          visibleGoods = visibleGoods.toReversed();
+        }
```

Comment:
It looks like you already did it on line 34, don't you? I'm wondering whether it doesn't affect the behavior and reverse the thing twice